RibonDAO/app
# Feat/prod3 487

# Description

- Adds minimum number of tickets information to non profits cards
- Adds new condition do disable button

## Does this pull request close any open issues?

- [#prod3-487](https://ribon.atlassian.net/browse/PROD3-487?atlOrigin=eyJpIjoiNzE5ZWJjMTc2NDE4NGEyYWI1ODUyNWE2MTM3ZjAxOWMiLCJwIjoiaiJ9)

## Type of change


## How to test

- Choose an active non profit
- edit the non profit impact to cost more than one ticket
- check if it is shown adequately at causes page
